bases/br_me_siconfi/despesa_funcao/table_config.yaml

```diff
@@ -0,0 +1,185 @@
+
+version: v1.0 # REQUIRED
+
+last_updated: 2021-08-11 # AUTO GENERATED
+
+# Descreva a tabela. Essas são as primeiras frases que um usuário vai ver.
+# Você não precisa ser muito conciso. Sinta-se a vontade para dar exemplos de
+# como usar os dados.
+# Se souber, liste também aplicações: pesquisa, apps, etc. que usem os dados.
+description: |  # REQUIRED
+   Despesas por função a nível de município.
+
+# Quem está completando esse arquivo config?
+published_by:
+    name: Crislane A. Souza  # REQUIRED
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_me_siconfi/code # REQUIRED
+    website: https://github.com/crislanealves
+    email: crislanealves@basedosdados.org
+
+# Qual organização/departamento/pessoa tratou os dados?
+# As vezes há um ponto intermediário entre os dados originais e subir na Base dos Dados.
+# Se essa pessoa é você, preencha abaixo com suas informações.
+treated_by:
+    name: Crislane A. Souza 
+    code_url: https://github.com/basedosdados/mais/tree/master/bases/br_me_siconfi/code 
+    website: <onde encontrar os dados tratados>
+    email: crislanealves@basedosdados.org
+
+# Se houve passos de tratamento, limpeza e manipulação de dados, descreva-os aqui.
+treatment_description: | 
+    - Criação da coluna 'id_municipio' para os anos de 1989-2012
+    - Criação da coluna 'id_conta_normalizado' para compatibilizar todos os anos que não tem 'portaria'
+    - Criação da coluna 'id_conta_bd' para compatilizar toda série temporal 
+    - Append nas tabelas dos anos 1996-2012 para a arquitetura 2013-2020    
+
+
+# Com qual frequência a base é atualizada?
+# Opções: hora | dia | semana | mes | 1 ano | 2 anos | 5 anos | 10 anos | unico | recorrente
+data_update_frequency: 1 ano # REQUIRED 
+
+# Nível da observação (qual é a granularidade de cada linha na tabela)
+# Escolha todas as opções necessárias.
+# Regras:
+#   - minúsculo, sem acento, singular.
+#   - em portugues (ou seja, não use os nomes de colunas abaixo)
+# Exemplos: pais, estado, municipio, cidade, hora, dia, semana, mes, ano, etc.
+observation_level: #REQUIRED
+    - municipio
+    - ano
+    - conta
+
+# Quais colunas identificam uma linha unicamente?
+# Preencha com os nomes de colunas. Ex: id_municipio, ano.
+# Pode ser vazio pois certas tabelas não possuem identificadores.
+primary_keys:
+    - id_municipo
+    - conta
+    - id_conta_bd
+
+# Qual é a cobertura espacial da tabela?
+# Regras:
+#   - minúsculo, sem acento, singular
+#   - descer até o menor nível administrativo cuja cobertura abaixo seja 'todos'
+# Exemplo 1: tabela que cubra todos os municípios nos estados de SP e GO
+#   - brasil
+#   - SP, GO
+# Exemplo 2: tabela que cubra países inteiros na América Latina
+#   - brasil, argentina, peru, equador
+coverage_geo:
+    - brasil
+
+# Qual é a cobertura temporal (em anos) da tabela?
+# Opções: ..., 1990, 1991, ..., 1999, 2000, 2001, ..., 2019, 2020, ...
+coverage_time:
+    - 1996
+    - 1997
```

Comment:
Os dados para 1996 and 1997 estão com qualidade boa? Caso não, talvez fosse melhor deixar a série começar em 2004 só. O que acha?